code/modules/mining/machine_processing.dm

```diff
@@ -4,14 +4,14 @@
 	name = "ore redemption console"
 	icon = 'icons/obj/machines/mining_machines.dmi'
 	icon_state = "console"
-	density = 0
-	anchored = 1
-	use_power = 1
+	density = FALSE
+	anchored = TRUE
+	use_power = TRUE
```

Comment:
Use Power is not a True/False Variable. See: code/game/machinery/machinery.dm Line 103-106